### PR TITLE
DEV-14896: Open port 16030 on region servers to enable HBase Metric collection

### PR DESCRIPTION
**Pull request by msoni-tamr**

### CODE REVIEW

File: modules/aws-emr-sgs/security-group-rules.tf [hunk 1]
```diff
@@ -539,6 +539,30 @@ resource "aws_security_group_rule" "region_server_port_add_core_sg_tamr_sgs" {
...
+  to_port           = 16030
+  protocol          = "tcp"
+  security_group_id = aws_security_group.emr_additional_core.id
+  type              = "ingress"
+  cidr_blocks       = var.tamr_cidrs
+  description       = "Region Server - HBase Metrics"
+}
+
+//Region Server rule - Additional SG for EMR Core with TAMR SGs for accessing HBase Metrics port
+resource "aws_security_group_rule" "region_server_port_add_core_sg_tamr_sgs" {
```
Comment (souza-dan) [inline modules/aws-emr-sgs/security-group-rules.tf:555]: Name is a duplicate from above.
https://github.com/Datatamer/terraform-aws-emr/pull/45/files#diff-31f8037b12d08f6f61a61459bde4dbda5aa8125ace0c1728a10c51c332d4ea2bR531

File: modules/aws-emr-sgs/security-group-rules.tf [hunk 2]
```diff
@@ -539,6 +539,30 @@ resource "aws_security_group_rule" "region_server_port_add_core_sg_tamr_sgs" {
   description              = "Region Server - EMR Dashboard"
 }
 
+//Region Server rule - Additional SG for EMR Core with TAMR CIDRS for accessing HBase Metrics port
+resource "aws_security_group_rule" "region_server_port_add_core_sg_tamr_cidrs" {
```
Comment (souza-dan) [inline modules/aws-emr-sgs/security-group-rules.tf:543]: Name is a duplicate of above. These resource names need to be unique.
https://github.com/Datatamer/terraform-aws-emr/pull/45/files#diff-31f8037b12d08f6f61a61459bde4dbda5aa8125ace0c1728a10c51c332d4ea2bR519

**Review by souza-dan — COMMENTED** [review]
Remember to update version and changelog. Seems like a minor version to me.

File: modules/aws-emr-sgs/security-group-rules.tf (hunk 1 above)
Comment (msoni-tamr) [reply modules/aws-emr-sgs/security-group-rules.tf]: Oops, good catch. Fixed.

File: modules/aws-emr-sgs/security-group-rules.tf (hunk 2 above)
Comment (msoni-tamr) [reply modules/aws-emr-sgs/security-group-rules.tf]: Oops, good catch. Fixed.

File: VERSION [hunk 3]
```diff
@@ -1 +1 @@
-4.0.0
+4.1.0
```
Comment (souza-dan) [inline VERSION:1]: Version is mentioned in comments in the examples:
https://github.com/msoni-tamr/terraform-aws-emr/blob/master/examples/static-hbase/main.tf#L47

May be worth searching `4.0.0` for any other uses.

**Review by souza-dan — APPROVED** [review]
LGTM

**Pull request merged**